sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_document_model_administration_client.py

```diff
@@ -207,7 +210,10 @@ def _compose_callback(raw_response, _, headers):  # pylint: disable=unused-argum
         if model_id is None:
             model_id = str(uuid.uuid4())
 
-        return self._client.begin_compose_document_model(  # type: ignore
+        _client_op_path = self._client
+        if self._api_version == DocumentAnalysisApiVersion.V2023_02_28_PREVIEW:
+            _client_op_path = _client_op_path.document_models
+        return _client_op_path.begin_compose_model(  # type: ignore
```

Comment:
same here, begin_compose_document_model for GA version and a few other places in this file where the 2022-08-31 name doesn't match the 2023-02 preview method name.